enterprise/server/remote_execution/execution_server/execution_server.go

```diff
@@ -686,6 +688,7 @@ func (s *ExecutionServer) execute(req *repb.ExecuteRequest, stream streamLike) e
 			if ee != "" {
 				ctx = log.EnrichContext(ctx, log.ExecutionIDKey, ee)
 				log.CtxInfof(ctx, "Reusing execution %q for execution request %q for invocation %q", ee, downloadString, invocationID)
+				tracing.AddStringAttributeToCurrentSpan(ctx, "task_id", "merged against "+ee)
```

Comment:
This can be useful when looking at trace, but not really while searching for one. Given that, I'm ambivalent on these being added.

@tylerwilliams thoughts on if these values should use this tag?

Reply:
Yeah, I toyed with the idea of just adding a new tag instead that shows which path the code took through here. Just pushed that version, LMK what you think.